### ISSUE TITLE
[Terraform 0.13] Pass Input Variable Validation

### ISSUE THREAD

**Issue by Tensho** [terraform-linters/tflint#890]
Terraform 0.13 switched [input variable custom validation rules](https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules) from **experimental** to **standard** features. However, `tflint` still want's this delacartion:

```hcl
terraform {
  experiments = [variable_validation]
}
```

### Example

```hcl
variable environment {
  type        = string
  description = "Environment"

  validation {
    condition     = can(index(["qa", "preprod", "prod"], var.environment))
    error_message = "The environment value must be \"qa\", \"preprod\" or \"prod\"."
  }
}
```

```
$ tflint variables.tf
Failed to load configurations. 1 error(s) occurred:

Error: Custom variable validation is experimental

  on ../module/variables.tf line 5, in variable "environment":
   5:   validation {

This feature is currently an opt-in experiment, subject to change in future releases based on feedback.

Activate the feature for this module by adding variable_validation to the list of active experiments.

```

### Expected Behaviour

Linter should pass variable validation for Terraform 0.13.

### Versions

TFLint version 0.18.0
Terraform v0.13.0

**Comment by wata727** [comment]
Did you try TFLint v0.19.1? v0.18.0 is compatible with Terraform v0.12.0, not v0.13.0.
See also https://github.com/terraform-linters/tflint/issues/881

**Comment by bendrucker** [comment]
Yes, as noted above and described in detail in #881, this is the most notable backwards incompatible change between Terraform 0.12 and 0.13. Configuration written against 0.13 features including non-experimental validation, for_each for modules, etc. require TFLint 0.19.
